geometric_features/aggregation/ocean/subbasin_regions.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python
+
+import shapely
+import numpy
+import xarray
+import os
+import matplotlib.pyplot as plt
+import pyproj
+import gzip
+import shutil
+import tempfile
+
+from geometric_features import GeometricFeatures, FeatureCollection
+from geometric_features.feature_collection import _round_coords
+
+from geometric_features.download import download_files
+from geometric_features.utils import write_feature_names_and_tags
```

Comment:
None of these imports are needed and they (`xarray` in particular) are causing trouble because they are not dependencies of `geometric_features`.  I'll be removing them in one of my commits.

Reply:
These have been removed in 626d423